components/server/src/orgs/organization-service.ts

```diff
@@ -327,11 +329,18 @@ export class OrganizationService {
 
     async getSettings(userId: string, orgId: string): Promise<OrganizationSettings> {
         await this.auth.checkPermissionOnOrganization(userId, "read_settings", orgId);
-        return (await this.teamDB.findOrgSettings(orgId)) || {};
+        const settings = (await this.teamDB.findOrgSettings(orgId)) || {};
+        if (!settings.defaultWorkspaceImage) {
+            settings.defaultWorkspaceImage = this.config.workspaceDefaults.workspaceImage;
```

Comment:
Would be awesome if we could roll this into a migration. :100:

Reply:
Not sure about it, since `config.workspaceDefaults.workspaceImage`. can be changeable after Gitpod installed (i.e. self-hosted).

So it's a bug that if there's we should allow undefined, partial org setting on dashboard and server.

Reply:
You are right: We also might want to change this on gitpod.io.

Reply:
Hey @geropl , just pushed a fixing for it https://github.com/gitpod-io/gitpod/pull/18723/commits/c2df2901c5f3a741f7f01b8d57916149d6abf770, could you take a look?

I'll do smoke test later today

Reply:
@geropl  With this commit https://github.com/gitpod-io/gitpod/pull/18723/commits/97ae60ffc83f179fa719af177348fbe84a89c120, it will allow user to submit empty string as default image, then we will store it with `NULL` in database

Reply:
Cool! We also have a [NULL_TO_UNDEFINED transformer](https://github.com/gitpod-io/gitpod/blob/7de05704b206cfaf482f9610c8c11ab7635f48ed/components/gitpod-db/src/typeorm/transformer.ts#L26) which Sven reported to be working.

(also, will leave you alone now with further bugging :bow: Thank you for all the effort!)

Reply:
Good to know it, will try it and smoke test 👍